src/main/java/mekanism/common/item/gear/ItemJetpack.java

```diff
@@ -89,18 +98,27 @@ public void setMode(ItemStack stack, JetpackMode mode) {
     @Override
     public void addHUDStrings(List<Component> list, Player player, ItemStack stack, EquipmentSlot slotType) {
         if (slotType == getSlot()) {
-            ItemJetpack jetpack = (ItemJetpack) stack.getItem();
-            list.add(MekanismLang.JETPACK_MODE.translateColored(EnumColor.DARK_GRAY, jetpack.getMode(stack)));
-            GasStack stored = GasStack.EMPTY;
-            Optional<IGasHandler> capability = stack.getCapability(Capabilities.GAS_HANDLER_CAPABILITY).resolve();
-            if (capability.isPresent()) {
-                IGasHandler gasHandlerItem = capability.get();
-                if (gasHandlerItem.getTanks() > 0) {
-                    stored = gasHandlerItem.getChemicalInTank(0);
-                }
+            addHUD(list, stack);
+        }
+    }
+
+    @Override
+    public void addCurioHUDStrings(List<Component> list, Player player, ItemStack stack) {
+        addHUD(list, stack);
```

Comment:
It would be cleaner (especially from a diff perspective) to just leave `addHUDStrings` alone and then make the implementation of this method just call `addHUDStrings(list, player, stack, getSlot())`